.github/workflows/pull_request.yml

```diff
@@ -58,6 +58,11 @@ jobs:
     - name: verify dependencies
       run: make deps
 
+    - name: verify kubegen
+      run: make verify_kubegen
+      env:
+        GOPATH: /home/runner/work/jiva-operator/go
```

Comment:
Do we need to  explicitly mention GOPATH here ?

Reply:
Yes I tried it without it and it failed. We are doing similar to this in zfs-localpv as well.